bundles/org.openhab.binding.magentatv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,258 @@
+			<channel id="stop" typeId="stop" />
+			<channel id="volumeUp" typeId="volumeUp" />
+			<channel id="volumeDown" typeId="volumeDown" />
+			<channel id="mute" typeId="system.mute" />
+			<channel id="channel" typeId="channel" />
+			<channel id="channelUp" typeId="channelUp" />
+			<channel id="channelDown" typeId="channelDown" />
+			<channel id="key" typeId="key" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="program">
+		<label>Program Information</label>
+		<description>Information on the running program</description>
+		<channels>
+			<channel id="title" typeId="system.media-title" />
+			<channel id="text" typeId="programText" />
+			<channel id="start" typeId="programStart" />
+			<channel id="duration" typeId="programDuration" />
+			<channel id="position" typeId="programPosition" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Play Status</label>
+		<description>Status information on media play</description>
+		<channels>
+			<channel id="channelCode" typeId="channelCode" />
+			<channel id="runStatus" typeId="runStatus" />
+			<channel id="playMode" typeId="playMode" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="stop">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops the player. ON if the player is stopped.</description>
+	</channel-type>
+	<channel-type id="channel">
+		<item-type>String</item-type>
+		<label>Channel</label>
+		<description>Send channel number to switch program</description>
+	</channel-type>
+	<channel-type id="channelCode" advanced="true">
+		<item-type>String</item-type>
+		<label>Channel Code</label>
+		<description>Channel code in the channel list</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="runStatus" advanced="true">
+		<item-type>String</item-type>
+		<label>Status</label>
+		<description>Run status</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="playMode">
+		<item-type>String</item-type>
+		<label>Play mode</label>
+		<description>Play mode for running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programTitle">
+		<item-type>String</item-type>
+		<label>Program</label>
+		<description>Running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programText">
+		<item-type>String</item-type>
+		<label>Description</label>
+		<description>Some info on the running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programStart">
+		<item-type>String</item-type>
+		<label>Start</label>
+		<description>Program start time</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programDuration">
+		<item-type>String</item-type>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.
```suggestion
		<item-type>Number:Time</item-type>
```

Reply:
done, various channels changed